make-boot-image/README.md

```diff
@@ -0,0 +1,35 @@
+# make-boot-image
+A oneshot service to download the specified Raspberry Pi linux-image- and
+create a replacement boot-image- package.  This replacement package contains a
+signed boot.img with a cryptroot-enabled initramfs. The kernel modules are
+retained in the replacement package. Necessary firmware file are inserted into
+the signed boot.img where appropriate (via raspi-firmware package).
+
+> [!CAUTION]
+> Support only exists for 2712 kernels at this time.
```

Comment:
Can this be made to default to 2711 kernels instead? We don't have official support for a 2712 platform at this time.

Reply:
I think we should probably default to kernel8.img which is the common 64-bit kernel.

kernel7l.img / (32-bit LPAE) is probably the wrong choice for new projects and the 2712 kernel with 16K images might be a micro-optimization for most projects.

Reply:
I'll update the PR to default to 2711/kernel8.img with raspi-firmare included.